next/components/molecules/SearchPageNew/AdvancedSearchNew.tsx

```diff
@@ -36,30 +37,34 @@ const AdvancedSearchNew = ({
 
   const newSearch = (
     <div className="flex flex-col items-start gap-8 self-stretch">
-      <h1 className="text-h1 font-semibold text-gray-800 lg:w-[800px]">{title}</h1>
+      <Typography type="h1" className="text-gray-800 lg:w-[800px]">
+        {title}
+      </Typography>
       <div className="flex w-full flex-col gap-y-2">
-        <SearchField
-          className="flex w-full flex-col [&>*>input>button]:hidden"
-          aria-label={t('search')}
-          defaultValue={placeholder}
-          value={input}
-          name="Searchinput"
-          onChange={(val) => {
-            return setInput(val)
-          }}
-          onSubmit={handleSearch}
-        >
-          <Text slot="description" className="mb-1 font-semibold">
+        <div className="flex flex-col gap-y-1">
+          <Typography type="p" className="font-semibold text-gray-800">
```

Comment:
Color `text-gray-800` should not be needed. 

Font weight `font-semibold` should be set by `fontWeight` prop of `Typography` component, not by `className`.